storage/innobase/xtrabackup/src/redo_log.h

```diff
@@ -271,6 +271,9 @@ class Archived_Redo_Log_Monitor {
   /** readiness flag. */
   std::atomic<bool> ready;
 
+  /** redo arch flag. */
```

Comment:
Elabore a bit more in the comment, also rename the variable name to something more useful. IIUC this flag controls if xtrabackup has set redo log arch